packages/for-ui/src/textField/TextField.tsx

```diff
@@ -117,12 +120,15 @@ export const TextField: React.ForwardRefExoticComponent<TextFieldProps> = React.
     );
 
     return (
-      <div className={fsx(['flex flex-col', className])}>
+      <div className={clsx(['flex  flex-col', className])}>
         <label>
           {label && (
             <p
-              className={fsx([
-                'text-s text-shade-medium-default mb-1 font-bold antialiased',
+              className={clsx([
+                'text-shade-medium-default mb-1 font-bold antialiased',
+                fontSize === undefined && 'text-s',
+                fontSize === 'large' && 'text-s',
+                fontSize === 'medium' && 'text-xs',
```

Comment:
```suggestion
                {
                  large: `text-r`,
                  medium: `text-s`,
                }[size]
```

でおねがいします！あとsizeはundefinedの場合はpropsでdefaultでlargeを仕込む感じでお願いしたいです！